DataFormats/FTLRecHit/interface/FTLRecHit.h

```diff
@@ -76,6 +100,8 @@ class FTLRecHit {
 private:
   DetId id_;
   float energy_, time_, timeError_;
+  std::pair<float, float> position_;
+  float positionError_;
```

Comment:
Hi,
I don't know too much about RecHits in ETL but from the parameters that passed into the constructor, it seems that BTL and ETL RecHits are using the same constructor but their properties are different because ETL don't have a pair of amplitude and time. Also, ETL don't have the position information.
Thanks.